packages/mail/src/classes/mail-service.js

```diff
@@ -110,6 +112,17 @@ class MailService {
     });
   }
 
+  /**
+   * Set default data to for all requests. Can be overwritten by incoming data on send
+   */
+  setDefaultData(data) {
+    if (Object.prototype.toString.call(data) !== "[object Object]") {
```

Comment:
What is this doing?

Reply:
Checking data is an object.

Reply:
Prefer to throw an exception if it's not an object instead of just returning.